Doc/c-api/contextvars.rst

```diff
@@ -101,6 +101,52 @@ Context object management functions:
    current context for the current thread.  Returns ``0`` on success,
    and ``-1`` on error.
 
+.. c:function:: int PyContext_AddWatcher(PyContext_WatchCallback callback)
+
+   Register *callback* as a context object watcher for the current interpreter.
+   Return an ID which may be passed to :c:func:`PyContext_ClearWatcher`.
+   In case of error (e.g. no more watcher IDs available),
+   return ``-1`` and set an exception.
+
+   .. versionadded:: 3.14
+
+.. c:function:: int PyContext_ClearWatcher(int watcher_id)
+
+   Clear watcher identified by *watcher_id* previously returned from
+   :c:func:`PyContext_AddWatcher` for the current interpreter.
+   Return ``0`` on success, or ``-1`` and set an exception on error
+   (e.g. if the given *watcher_id* was never registered.)
+
+   .. versionadded:: 3.14
+
+.. c:type:: PyContextEvent
+
+   Enumeration of possible context object watcher events:
+   - ``PY_CONTEXT_EVENT_ENTER``
+   - ``PY_CONTEXT_EVENT_EXIT``
```

Comment:
```suggestion
   - ``Py_CONTEXT_EVENT_ENTER``
   - ``Py_CONTEXT_EVENT_EXIT``
```

Reply:
The other watcher enum are all caps

Reply:
Yeah, those went under the radar. cc. @encukou

Reply:
Yes, they did.
You can be consistent with them or follow PEP-7. I don't think it's worth starting a heated discussion over this ;)

Reply:
I think i'm going to go with matching the existing enum style for consistency with this class of features.  And if I have time i'll issue a pull to add the correct PEP-7 names to each of the enums so we can transition at some future date to using the correct terms.  I feel like that change might be contentious, but I feel all of them should match up.